static/js/controllers/configuration-forms-xml.js

```diff
@@ -59,7 +59,14 @@ angular.module('inboxControllers').controller('ConfigurationFormsXmlCtrl',
           var meta = results[1];
 
           var $xml = $($.parseXML(xml));
+
           var title = $xml.find('title').text();
+          if (!title) {
```

Comment:
Maybe we should only use the regex version if the `$xml.find` doesn't work in modern Chrome? Do we know why it doesn't work?

Reply:
I think it's to do with XML namespaces.  But yes, it would be nice to understand _why_!